src/components/layout/PetCard.tsx

```diff
@@ -44,6 +45,7 @@ export default function PetCard(props: PetCardType) {
           <LinkContainer to={`/animal/${id}`}>
             <Button className="card__btn">More Info</Button>
           </LinkContainer>
+          <FavoriteSection id={id} />
```

Comment:
Home page uses this petcard  too and all the users access it. 

You can Add a button in `Favorites.tsx` where pet List map is 
```
          {petList.map((pet) => (
            <><PetCard
              key={pet.id}
              breeds={pet.breeds}
              id={pet.id}
              name={pet.name}
              photos={pet.photos}
              type={pet.type}
              primary_photo_cropped={pet.primary_photo_cropped}
            />  
// some button with remove favorites functionality
</>
          ))}
```

Reply:
the remove button onlclick would have something similar to function

```
  const removeFav = async () => {
    await executeDelete((query) => query.eq("id", removalId), {
      returning: "minimal",
      count: "estimated",
    });
    dispatch(removeFavoritePet(removalId));
  };
```